Build.ps1

```diff
@@ -213,4 +210,5 @@ SitecoreImageBuilder\Invoke-Build `
     -InstallSourcePath $InstallSourcePath `
     -Registry $Registry `
     -Tags $tags `
+    -ExperimentalTagBehavior:(@{$true="Include";$false="Skip"}[$IncludeExperimental -eq $true]) `
```

Comment:
I'll remember this snippet - had no idea this could be done

Reply:
Basically indexing into a hashtable.